README.adoc

```diff
@@ -20,139 +21,219 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/how_to_complete_this_guide.adoc[]
 
-
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/hide-show-gradle.adoc[]
-
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/hide-show-maven.adoc[]
-
-include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/master/hide-show-sts.adoc[]
-
 == Business Data
 
-Typically your customer or a business analyst supplies a spreadsheet. In this case, you make it up.
+Typically, your customer or a business analyst supplies a spreadsheet. For this simple
+example, you can find some made-up data in `src/main/resources/sample-data.csv`:
 
-`src/main/resources/sample-data.csv`
+====
 [source,csv]
 ----
 include::initial/src/main/resources/sample-data.csv[]
 ----
+====
 
-This spreadsheet contains a first name and a last name on each row, separated by a comma. This is a fairly common pattern that Spring handles out-of-the-box, as you will see.
-
+This spreadsheet contains a first name and a last name on each row, separated by a comma.
+This is a fairly common pattern that Spring can handle without customization.
 
-Next, you write a SQL script to create a table to store the data.
+Next, you need to write an SQL script to create a table to store the data. You can find
+such a script in `src/main/resources/schema-all.sql`:
 
-`src/main/resources/schema-all.sql`
+====
 [source,sql]
 ----
 include::initial/src/main/resources/schema-all.sql[]
 ----
+====
 
 NOTE: Spring Boot runs `schema-@@platform@@.sql` automatically during startup. `-all` is the default for all platforms.
 
+== Starting with Spring Initializr
+
+For all Spring applications, you should start with the https://start.spring.io[Spring
+Initializr]. The Initializr offers a fast way to pull in all the dependencies you need for
+an application and does a lot of the set up for you. This example needs the Spring Batch
+and HyperSQL Database dependencies. The following image shows the Initializr set up for
+this sample project:
+
+image::images/initializr.jpg[]
+
```

Comment:
`.png`?

Reply:
Yessir. I first made a screenshot with the Macbook's utilities, but it doesn't work for the whole screen. So I added an extension that does make a whole screen screenshot. The Macbook saved as JPG. The extension saves as PNG. Then I copy/pasted that .jpg all over the place.

Sorry about the mess. I'll have it fixed in all those projects in a few minutes.